tdrs-backend/tdpservice/users/test/test_auth_check.py

```diff
@@ -1,9 +1,13 @@
 """Test the authorization check."""
 
 import pytest
+import logging
```

Comment:
We don't seem to be using logging in `test_auth_check` other than setting it to `INFO`. Can we take it out?